README.md

```
@@ -1,38 +1,189 @@
+
+### Vocabulary Importer
+
+<p float="left">
+  <img src="homework-screenshots/7_vocab-importer-dark-mode.png" width="400" />
+  <img src="homework-screenshots/8_vocab-importer-light-mode.png" width="400" /> 
+</p>
+
+### Arabic Listening Comprehension
+
+<p float="left">
+  <img src="homework-screenshots/9_listening_arabic-quiz-app.png" width="400" />
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)